### PR TITLE
#5731 [Manager] Replace time consuming cypress commands with API based commands

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Review by saivirtue — COMMENTED** [review]
Just wondering, does this PR covers the change of volume flow?

**Comment by konekoya** [comment]
> Just wondering, does this PR covers the change of volume flow?

Not yet. I will add new tests for volume after the new UI is added :)

File: ohara-manager/client/cypress/global.d.ts [hunk 1]
```diff
@@ -29,35 +33,43 @@ declare namespace Cypress {
...
      * Get the _&lt;td /&gt;_ elements by required parameters.
      * <p> This function has the following combination:
-     *
-     * <p> 1. `columnName`: filter all _&lt;td /&gt;_ elements of specific column.
-     *
-     * <p> 2. `columnName + columnValue`: filter the _&lt;td /&gt;_ element of specific column and value.
-     *
-     * <p> 3. `columnName + rowFilter`: filter the _&lt;td /&gt;_ element of specific column in specific rows.
-     *
-     * <p> 4. `columnName + columnValue + rowFilter`: filter the _&lt;td /&gt;_ element of specific column in specific rows.
```
Comment (konekoya) [inline ohara-manager/client/cypress/global.d.ts:47]: These examples won't show up in VSCode's tooltip unless we place them in the `@example` keyword 🤷‍♂️

Comment (saivirtue) [reply ohara-manager/client/cypress/global.d.ts:47]: Hmm...good to know this 👍 
Then why we don't use the `@example` keyword instead?

Comment (konekoya) [reply ohara-manager/client/cypress/global.d.ts:47]: We did, they are now listed with `@example` :)

https://github.com/oharastream/ohara/pull/5736/files#diff-e45d3cbc638ae98a6076482d3519556b5c55e9e9224aa767e78dd57869d4967dR62-R65

**Comment by konekoya** [comment]
retry manager-e2e

**Review by saivirtue — APPROVED** [review]

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**